pkg/aws/ami.py

```diff
@@ -17,86 +17,98 @@ async def start_ami_creation_worker(bucket_name, image_name, project, disk_conta
    secret_key = Project.objects(name=project).allow_filtering()[0]['secret_key']
    region = Project.objects(name=project).allow_filtering()[0]['location']
    import_task_id = ''
+   role_id = "vmimport"
    try:
       client = boto3.client('iam', aws_access_key_id=access_key, aws_secret_access_key=secret_key)
-      file_trust_policy='''{
-         "Version":"2012-10-17",
-         "Statement":[
-            {
-               "Sid":"",
-               "Effect":"Allow",
-               "Principal":{
-                  "Service":"vmie.amazonaws.com"
-               },
-               "Action":"sts:AssumeRole",
-               "Condition":{
-                  "StringEquals":{
-                     "sts:ExternalId":"vmimport"
+
+      role_list = [role['RoleName'] for role in client.list_roles()['Roles']]
+      if role_id not in role_list:
+         file_trust_policy={
+            "Version": "2012-10-17",
+            "Statement": [
+               {
+                  "Effect": "Allow",
+                  "Principal": { "Service": "vmie.amazonaws.com" },
+                  "Action": "sts:AssumeRole",
+                  "Condition": {
+                     "StringEquals":{
+                        "sts:Externalid": 'vmimport'
```

Comment:
u will also need to update the dynamic name of the role here too.

Reply:
This one can only be vmimport, it is like a default value. The role will not work if this is changed.